### ISSUE TITLE
Add an integration with Splunk and/or AWS GuardDuty

### ISSUE THREAD

**Issue by DiegoVazquezNanini** [alerta/alerta#1488]
**Is your feature request related to a problem? Please describe.**
Add all the goodness of Alerta to Security Tools.

**Describe the solution you'd like**
Integrate against Splunk and AWS GuardDuty APIs.

**Additional context**
This will make Alerta relevant to Security Operation Teams.

I hope you are well @satterly

**Comment by DiegoVazquezNanini** [comment]
Hi @3IMOH,
Apologies if my submission was not very clear. Let me see if I can amend that.
My suggestion was around making alerta integrate with the various AWS notifications/alarms. (or equivalent systems from GCP and Azure). It would be really useful if you can get all the CloudWatch Alarms, GuardDuty, Billing notifications, SNS topics, etc into the alerta dashboard.

The general point I'm trying to make is that this great tool could be made more general by adding other types of alarms/notifications. (for example, for security operations teams integrating with the above APIs or products like Splunk)

Feel free to close the issue, if you think this is not appropriate. I'm happy to provide links to the various APIs if others believe this is a good idea.

Kind regards

**Comment by satterly** [comment]
Yes, please provide links to relevant APIs. Even if we decide not to work on it in the near future it would be easier to pick up by someone when they have time. Thanks.

I hope you and the family are well. 😄

**Comment by satterly** [comment]
Closing due to inactivity.
